llvm/test/CodeGen/AMDGPU/mfma-loop.ll

```diff
@@ -2435,39 +2435,40 @@ define amdgpu_kernel void @test_mfma_nested_loop_zeroinit(ptr addrspace(1) %arg)
 ;
 ; GFX942-LABEL: test_mfma_nested_loop_zeroinit:
```

Comment:
Materializes `0` as b64 which is done through VGPR with v_mov_b64, results in regression of +1 instruction instead of materializing into a0 and re-using that to set the remainder.